root/account/edit.tt

```diff
@@ -65,8 +65,8 @@
             </span>
             [% extra_field = form.field('languages').field(form.field('languages').add_extra(1) - 1) %]
             <span id="add-language-template" style="display: none">
-              [% r.select(extra_field.field('language_id'), id => 'fluency-template', class => 'language_id',) %]
-              [% r.select(extra_field.field('fluency'), class => 'fluency') %]
+              [% r.select(extra_field.field('language_id'), id => 'id-language-id-template', class => 'language_id', name => 'language-id-template') %]
+              [% r.select(extra_field.field('fluency'), id => 'id-fluency-template', class => 'fluency', name => 'fluency-template') %]
```

Comment:
Indeed `id`s are of no use. They had to be unique, hence the override, but removal is even better.
Only `name`s are needed for form submission. And yes, template’s `name` is worthless.

Cleaned up in https://github.com/metabrainz/musicbrainz-server/pull/472/commits/547b27e529989a708b1e61616f40fde7bc6736b3